pkg/api/testing/fuzzer.go

```diff
@@ -112,7 +112,7 @@ func FuzzerFor(t *testing.T, version string, src rand.Source) *fuzz.Fuzzer {
 			*j = statuses[c.Rand.Intn(len(statuses))]
 		},
 		func(j *api.PodTemplateSpec, c fuzz.Continue) {
-			// TODO: v1beta1/2 can't round trip a nil template correctly, fix by having v1beta1/2
+			// TODO: v1beta1/2(deleted on June 1, 2015) can't round trip a nil template correctly, fix by having v1beta1/2(deleted on June 1, 2015)
```

Comment:
If the comment is true, it should still pass tests when you remove this fuzz function completely?